packages/iTwinUI-react/src/core/utils/functions/dom.ts

```diff
@@ -38,3 +38,27 @@ export const getDocument = () => {
 export const getWindow = () => {
   return typeof window === 'undefined' ? undefined : window;
 };
+
+export const getScrollableParent = (
+  element: HTMLElement | undefined | null,
+  ownerDocument: Document = document,
+): HTMLElement => {
+  if (!element || element === ownerDocument.body) {
+    return ownerDocument.body;
+  }
+
+  return isElementScrollable(element)
+    ? element
+    : getScrollableParent(element.parentElement, ownerDocument);
+};
+
+export const isElementScrollable = (element: HTMLElement) => {
+  return /(auto|scroll|overlay)/.test(
+    getElementStyle(element, 'overflow') +
+      getElementStyle(element, 'overflow-y'),
```

Comment:
Should we change this function to just compare `scrollHeight`/`offsetHeight`?

@veekeys Was there any reason you went with this logic instead of heights?

Reply:
Hm... SOmething was not working with heights, I think...
What is the issue with this one?

Reply:
@bentleyvk had to create this same function with heights comparison, because overflow does not automatically imply scrolling. But not relevant anymore.